src/index.js

```diff
@@ -46,6 +46,11 @@ class TracePlugin {
       'post:setup': this.postSetup.bind(this),
       'pre:report': this.preReport.bind(this)
     };
+    this.meta = {
+      name: 'trace',
+      version: '0.1.2',
```

Comment:
@kolanos would you mind making a unit test that compares this value to the package.json value so we always keep them in line?